app/services/account_reset_health_checker.rb

```diff
@@ -3,16 +3,16 @@ module AccountResetHealthChecker
 
   # @return [HealthCheckSummary]
   def check
-    rec = find_request_not_serviced_within_26_hours
-    HealthCheckSummary.new(healthy: rec.nil?, result: rec)
+    unserviced_request_exists = request_not_serviced_within_26_hours?
+    HealthCheckSummary.new(healthy: !unserviced_request_exists, result: unserviced_request_exists)
   end
 
   # @api private
-  def find_request_not_serviced_within_26_hours
+  def request_not_serviced_within_26_hours?
```

Comment:
Since we're here, if the wait period is configurable, can we avoid the magic number in the method name?

Something like...

```suggestion
  def request_not_serviced_within_wait_period?
```

...or, to be more precise:

```suggestion
  def request_not_serviced_within_wait_period_plus_2_hours?
```

Reply:
Yes, good call